src/styles/nav.scss

```diff
@@ -0,0 +1,38 @@
+.nav {
+  cursor: pointer;
```

Comment:
Consider adding a transition to make the hover effect smooth as stated in the task description. For example, add 'transition: color 0.3s ease;' to the .nav__link class.